<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sync v1.2.1</string>
  </property>
  <property name="windowIcon">
   <iconset resource="application.qrc">
    <normaloff>:/Icon/Sync.png</normaloff>:/Icon/Sync.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit_Source">
    <property name="geometry">
     <rect>
      <x>299</x>
      <y>10</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Source">
    <property name="geometry">
     <rect>
      <x>649</x>
      <y>10</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Choisir le dossier source</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="application.qrc">
      <normaloff>:/Icon/Dossier.png</normaloff>:/Icon/Dossier.png</iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Destination">
    <property name="geometry">
     <rect>
      <x>299</x>
      <y>60</y>
      <width>341</width>
      <height>30</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Destination">
    <property name="geometry">
     <rect>
      <x>649</x>
      <y>60</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Choisir le dossier destination</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="application.qrc">
      <normaloff>:/Icon/Dossier.png</normaloff>:/Icon/Dossier.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Echo">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>100</y>
      <width>61</width>
      <height>46</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Echo</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="application.qrc">
      <normaloff>:/Icon/Echo.png</normaloff>:/Icon/Echo.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Sync">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>100</y>
      <width>61</width>
      <height>46</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Syncronisation</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="application.qrc">
      <normaloff>:/Icon/Syncro.png</normaloff>:/Icon/Syncro.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Contri">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>100</y>
      <width>61</width>
      <height>46</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Contribution</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="application.qrc">
      <normaloff>:/Icon/Contribution.png</normaloff>:/Icon/Contribution.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Quitter">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>315</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>285</y>
      <width>470</width>
      <height>25</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Lancer">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>315</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Lancer</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_Resume">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>450</width>
      <height>85</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::DoubleClicked</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>134</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>134</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>30</number>
    </attribute>
    <row>
     <property name="text">
      <string>Dossier</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Fichier</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Ajout source</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ajout destination</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Suppr destination</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>0</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>0</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>0</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>0</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>0</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>0</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_Wait">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>320</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Veuillez patienter...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_source">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>13</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Source :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_destination">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Destination :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_fichierEnCours">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>471</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_profils">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Profils</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QListWidget" name="listWidget_profils">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>241</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_nouveauProfil">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Créer un profil</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="application.qrc">
       <normaloff>:/Icon/Plus.png</normaloff>:/Icon/Plus.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_supprimerProfil">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>290</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Supprimer le profil</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="application.qrc">
       <normaloff>:/Icon/Supprimer.png</normaloff>:/Icon/Supprimer.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nomProfil">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>265</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_annulerProfil">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>265</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="application.qrc">
       <normaloff>:/Icon/Annuler.png</normaloff>:/Icon/Annuler.png</iconset>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_Quitter</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>729</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_Source</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_verifChamps()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>469</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_Destination</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_verifChamps()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>469</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Contri</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>analyser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Sync</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>analyser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Echo</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>analyser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_verifChamps()</slot>
  <slot>analyser()</slot>
 </slots>
</ui>
